src/app/qgsoptions.cpp

```diff
@@ -782,11 +782,11 @@ QgsOptions::QgsOptions( QWidget *parent, Qt::WindowFlags fl, const QList<QgsOpti
   setZoomFactorValue();
 
   // predefined scales for scale combobox
-  QString myPaths = mSettings->value( QStringLiteral( "Map/scales" ), PROJECT_SCALES ).toString();
-  if ( !myPaths.isEmpty() )
+  QString scalePaths = mSettings->value( QStringLiteral( "Map/scales" ), PROJECT_SCALES ).toString();
+  if ( !scalePaths.isEmpty() )
   {
-    QStringList myScalesList = myPaths.split( ',' );
-    Q_FOREACH ( const QString &scale, myScalesList )
+    QStringList ScalesList = scalePaths.split( ',' );
```

Comment:
watch out, variable names first character is lower-case (class names are capital first letter)

Reply:
and also watch out: if using this `for ( : )` construct, the container part needs to be `const`.

Reply:
> if using this for ( : ) construct, the container part needs to be const.

That means that i should rename the other occurrences of pathList, ie for svg, composer templates, hidden path...?

Reply:
Easiest: `const QStringList scalesList = ...`, with the implication that different names will need to be used for different variables (which I think makes sense, they *are* different lists after all).
Or: `for ( const T &var : qgis::as_const( container ) )`

caveat: there are also other rules in other places (like using member variables within const methods or using `qgis::as_const` and then modifying the value of the original container).